docs/search_vignette.ipynb

```diff
@@ -155,23 +155,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## \\_\\_find_add_srows\n",
+    "## \\_\\_find_add_rows\n",
     "\n",
-    "`__find_add_srows` finds pairs of repeated structures, representated as diagonals of a certain length that start at the same time step as previously found pairs of repeated structures of the same length. \n",
+    "`__find_add_rows` Finds pairs of repeated structures, represented as diagonals of a certain\n",
+    "    length, k, that start at, end at, or in thr middle of the same time steps\n",
+    "    as previously found pairs of repeated structures of the same length.\n",
     "\n",
-    "The inputs for the function are: \n",
+    "The inputs for the functions are: \n",
     "\n",
-    "- __lst_no_anno__ (np.ndarray): Pairs of repeats      \n",
-    "- __check_inds__ (np.ndarray): List of starting indices of repeats \n",
-    "- __k__ (int): Length of repeat that we are looking for\n",
+    "- __lst_no_anno__ (np.ndarray): Pairs of repeats\n",
+    "- __check_inds__ (np.ndarray): List of starting indices of repeats\n",
+    "- __k__ (int): Length of repeats that we are looking for \n",
```

Comment:
I think "repeat" should be singular (w/o "s") because I'm seeing it as just another way of saying "repeat length"